cmd/vic-machine/configure/configure.go

```diff
@@ -85,6 +105,59 @@ func (c *Configure) Flags() []cli.Flag {
 			Destination: &c.upgrade,
 			Hidden:      true,
 		},
+		cli.BoolFlag{
```

Comment:
flags should be generated in common/certificate.go, so that can be reused by both create and configure command